src/etc/inc/services.inc

```diff
@@ -181,6 +181,17 @@ function services_radvd_configure($blacklist = array()) {
 				$radvdconf .= "\t\tAdvRouterAddr on;\n";
 				break;
 		}
+
+		$validlifetime = $dhcpv6ifconf['ravalidlifetime'];
+		if(is_numeric($validlifetime)) {
+		  $radvdconf .= "\t\tAdvValidLifetime {$validlifetime};\n";
```

Comment:
IMO you don't need to set $validlifetime here, you can check using array value direct and using is_numericint() to make sure it's a positive number
```
if (is_numericint($dhcpv6ifconf['ravalidlifetime'])) {
```